src/MongoDB.Driver.Core/Core/Clusters/ServerSelectors/CompositeServerSelector.cs

```diff
@@ -47,6 +47,11 @@ public IEnumerable<ServerDescription> SelectServers(ClusterDescription cluster,
             foreach (var selector in _selectors)
             {
                 selectedServers = selector.SelectServers(cluster, selectedServers);
+
```

Comment:
I would say these two lines(49, 51) are related to each other, so consider to remove the blank line (50) between them. But I'm ok to leave it if you see a sense in it

Reply:
Done